.env.template

```
@@ -7,3 +7,9 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY=''
+
+TRELLO_API_TOKEN=''
+
+TRELLO_BOARD_ID=''
```

I think we're missing two variables here
```suggestion
TRELLO_BOARD_ID=

TO_DO_LIST_ID=

DONE_LIST_ID
```